share/completions/figlet.fish

```diff
@@ -0,0 +1,29 @@
+function __fish_print_figlet_fonts
+    find (figlet -I2) -type f ^ /dev/null | sed -e 's$/.*/\([^/.]*\)\.[ft]lf$\1\tFont$;tx;d;:x'
```

Comment:
How about

```fish
set -l dir (figlet -I 2)
set -l files $dir/*.flf $dir/*.tlf

printf '%s\tFont\n' (string replace -r '.*/([^/]+)\.[ft]lf' '$1' -- $files)
```

?

You might want to test the performance here. To do that, define the function, execute it and `echo $CMD_DURATION` right after. That'll give you the time in milliseconds.